CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   * option --update has been added - sets percentage of update operations in bench space.
   * option --fill" has been added - sets number of records to pre-fill the space.
 
+### Fixed
+
+- Fixed sporadic "Failed to dial" errors on cartridge cli different commands (#697).
```

Comment:
This is for releasenotes, you don't need a ticket number. You can add a clarification that the connection to the local instance will now be used.

Reply:
Done.